airgun/entities/host_new.py

```diff
@@ -23,6 +27,85 @@ def get_details(self, entity_name, widget_names=None):
         view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
         return view.read(widget_names=widget_names)
 
+    def get_packages(self, entity_name, search=""):
+        """Filter installed packages on host"""
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.Content.Packages.select()
```

Comment:
`Content` and `Packages` are `Tab` classes, this `select()` is redundant, `Tab` implements `child_widget_accessed` and clicks itself. You should be able to directly call `searchbar.fill`

Reply:
Yeah I thought so, but without this line, it was actually not able to locate everything on that page for some reason

Reply:
Started convo about this in chat as it could take some debug - starting with suggestion to not set ROOT for tab classes.